pennylane/devices/default_qubit.py

```diff
@@ -247,15 +263,26 @@ def execute(
         )
         if max_workers is None:
             results = tuple(
-                simulate(c, rng=self._rng, debugger=self._debugger, interface=interface)
+                simulate(
+                    c,
+                    rng=self._rng,
+                    prng_key=self._prng_key,
+                    debugger=self._debugger,
+                    interface=interface,
+                )
                 for c in circuits
             )
         else:
             vanilla_circuits = [convert_to_numpy_parameters(c) for c in circuits]
             seeds = self._rng.integers(2**31 - 1, size=len(vanilla_circuits))
             _wrap_simulate = partial(simulate, debugger=None, interface=interface)
             with concurrent.futures.ProcessPoolExecutor(max_workers=max_workers) as executor:
-                exec_map = executor.map(_wrap_simulate, vanilla_circuits, seeds)
+                exec_map = executor.map(
+                    _wrap_simulate,
+                    vanilla_circuits,
+                    seeds,
+                    [self._prng_key] * len(vanilla_circuits),
+                )  # does this not need interface and debugger for this part?
```

Comment:
interface will strictly be numpy, as backpropagation falls over with multiprocessing. Debugging is also incompatible with multiprocessing.

Reply:
Ah, thanks! (Also for reminding me I need to remove these comments 😅 )